pkg/api/validation/register.go

```diff
@@ -1,8 +1,6 @@
 package validation
 
 import (
-	_ "github.com/openshift/origin/pkg/api/install"
```

Comment:
It's imported twice which my IDE complains about / gets a bit confused by.  See the last import in that block.  Or perhaps there was some nuanced Go reason for doing this?